### PR TITLE
Add dconf_gnome_disable_automount to RHEL STIG profile.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Select rule to STIG profile
- Update references section
- Add RHEL8 cce
- Enable remediation for Fedora
- Change severity to medium as defined by DISA STIG
#### Rationale:

- Rule is part of newer STIG: https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_7_security_technical_implementation_guide/V-100023?version=v2r7

Update1: Rule has been broken down into three rules, each of them checks for a specific parameter in gnome dconf configuration.

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by mildas** [comment]
**Changes identified:**
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by mildas** [comment]
**Changes identified:**
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by mildas** [comment]
**Changes identified:**
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by vojtapolasek** [comment]
I am trying to run tests for this rule with upstream test suite and VM and the rule errors out as "not applicable". I see there is
platform: machine
Is that needed? Is this the reason why the test reports "not applicable"?

**Comment by ggbecker** [comment]
> I am trying to run tests for this rule with upstream test suite and VM and the rule errors out as "not applicable". I see there is
> platform: machine
> Is that needed? Is this the reason why the test reports "not applicable"?

You need to have `dconf` installed in your testing VM.

**Comment by redhatrises** [comment]
This rule needs to be broken into at least 3 separate rules.

**Comment by vojtapolasek** [comment]
> This rule needs to be broken into at least 3 separate rules.

What is your reasoning? Would it make sense to have in a profile for example only two of these three config variables configured?

**Comment by redhatrises** [comment]
> > This rule needs to be broken into at least 3 separate rules.
> 
> What is your reasoning? Would it make sense to have in a profile for example only two of these three config variables configured?

Yes. It would. It also produces better security content.

**Comment by ggbecker** [comment]
> > > This rule needs to be broken into at least 3 separate rules.
> > 
> > 
> > What is your reasoning? Would it make sense to have in a profile for example only two of these three config variables configured?
> 
> Yes. It would. It also produces better security content.

Ok, I'll add this to my backlog and will get back to it soon.

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_disable_automount:
 Template usage changed in ansible remediation.
 Attribute value changed in OVAL check.
 Ansible remediation changed.
 Node moved within OVAL check.
 Text changed in OVAL check.
 Deleted attribute from OVAL check.
 Node deleted from OVAL check.
Rule dconf_gnome_disable_automount_open:
 OVAL check is newly added.
 Ansible remediation newly added.
 Bash remediation is newly added.
Rule dconf_gnome_disable_autorun:
 OVAL check is newly added.
 Ansible remediation newly added.
 Bash remediation is newly added.
Profile ncp on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to ncp profile.
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_automount, dconf_gnome_disable_autorun added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by ggbecker** [comment]
> > > > This rule needs to be broken into at least 3 separate rules.
> > > 
> > > 
> > > What is your reasoning? Would it make sense to have in a profile for example only two of these three config variables configured?
> > 
> > 
> > Yes. It would. It also produces better security content.
> 
> Ok, I'll add this to my backlog and will get back to it soon.

@redhatrises rule has been broken down into three rules. Can you verify them?

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_disable_automount:
 Deleted attribute from OVAL check.
 Node moved within OVAL check.
 Attribute value changed in OVAL check.
 Ansible remediation changed.
 Text changed in OVAL check.
 Node deleted from OVAL check.
 Template usage changed in ansible remediation.
Rule dconf_gnome_disable_automount_open:
 Bash remediation is newly added.
 Ansible remediation newly added.
 OVAL check is newly added.
Rule dconf_gnome_disable_autorun:
 Bash remediation is newly added.
 Ansible remediation newly added.
 OVAL check is newly added.
Profile ncp on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to ncp profile.
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount, dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_disable_automount:
 Ansible remediation changed.
 Deleted attribute from OVAL check.
 Attribute value changed in OVAL check.
 Text changed in OVAL check.
 Node moved within OVAL check.
 Node deleted from OVAL check.
 Template usage changed in ansible remediation.
Rule dconf_gnome_disable_automount_open:
 Ansible remediation newly added.
 Bash remediation is newly added.
 OVAL check is newly added.
Rule dconf_gnome_disable_autorun:
 Ansible remediation newly added.
 Bash remediation is newly added.
 OVAL check is newly added.
Profile ncp on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to ncp profile.
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount, dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by ggbecker** [comment]
@redhatrises can you check this one?

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_disable_automount:
 Attribute value changed in OVAL check.
 Ansible remediation changed.
 Node moved within OVAL check.
 Deleted attribute from OVAL check.
 Node deleted from OVAL check.
 Template usage changed in ansible remediation.
 Text changed in OVAL check.
Rule dconf_gnome_disable_automount_open:
 OVAL check is newly added.
 Bash remediation is newly added.
 Ansible remediation newly added.
Rule dconf_gnome_disable_autorun:
 OVAL check is newly added.
 Bash remediation is newly added.
 Ansible remediation newly added.
Profile ncp on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to ncp profile.
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount, dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by redhatrises** [comment]
Looks good except for the duplicate STIGIDs. We should remove them, and ask for new ones.

**Comment by ggbecker** [comment]
> Looks good except for the duplicate STIGIDs. We should remove them, and ask for new ones.

I assume that it can take a while until they (DISA) split the rules and assign new STIG ids to them. So, do we want to wait for that? And how do we initiate this request to DISA?

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<def-group>
+  <definition class="compliance" id="dconf_gnome_disable_automount_open" version="1">
+    <metadata>
+      <title>Disable GNOME3 automount-open</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml:6]: I would use the Jinja macro here.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount/oval/shared.xml [hunk 2]
```diff
@@ -1,27 +1,22 @@
 <def-group>
-  <definition class="compliance" id="dconf_gnome_disable_automount" version="1">
+  <definition class="compliance" id="dconf_gnome_disable_automount" version="2">
     <metadata>
-      <title>Disable GNOME3 Automounting</title>
+      <title>Disable GNOME3 automount</title>
       <affected family="unix">
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount/oval/shared.xml:5]: I would use Jinja macro here.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<def-group>
+  <definition class="compliance" id="dconf_gnome_disable_autorun" version="1">
+    <metadata>
+      <title>Disable GNOME3 Automounting</title>
+      <affected family="unix">
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml:5]: I would use Jinja macro here.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="Disable automount-open in GNOME"
+  id="test_dconf_gnome_disable_automount_open" version="1">
+    <ind:object object_ref="obj_dconf_gnome_disable_automount_open" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_dconf_gnome_disable_automount_open"
+  version="1">
+    <ind:path>/etc/dconf/db/local.d/</ind:path>
+    <ind:filename operation="pattern match">^.*$</ind:filename>
+    <ind:pattern operation="pattern match">^\[org/gnome/desktop/media-handling]([^\n]*\n+)+?automount-open=false$</ind:pattern>
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml:34]: I see that the  closing bracket is not escaped... it is not causing any problems now but I would do that. Please see also other ovals.
```suggestion
    <ind:pattern operation="pattern match">^\[org/gnome/desktop/media-handling\]([^\n]*\n+)+?automount-open=false$</ind:pattern>
```

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="Disable autorun in GNOME"
+  id="test_dconf_gnome_disable_autorun" version="1">
+    <ind:object object_ref="obj_dconf_gnome_disable_autorun" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_dconf_gnome_disable_autorun"
+  version="1">
+    <ind:path>/etc/dconf/db/local.d/</ind:path>
+    <ind:filename operation="pattern match">^.*$</ind:filename>
+    <ind:pattern operation="pattern match">^\[org/gnome/desktop/media-handling]([^\n]*\n+)+?autorun-never=true$</ind:pattern>
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml:34]:
```suggestion
    <ind:pattern operation="pattern match">^\[org/gnome/desktop/media-handling\]([^\n]*\n+)+?autorun-never=true$</ind:pattern>
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Looks very good, just small fixes needed, please see comments.

**Comment by vojtapolasek** [comment]
Oh, while running SST test suite, unfortunately none of ansible remediations seem to pass the test. Bash is ok.

**Comment by ggbecker** [comment]
> Oh, while running SST test suite, unfortunately none of ansible remediations seem to pass the test. Bash is ok.

I'll eventually get back to this pull request when I have time.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml (hunk 5 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml]: Done

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml (hunk 4 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml]: Done

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml (hunk 3 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/oval/shared.xml]: Used `oval_metadata` macro.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount/oval/shared.xml (hunk 2 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount/oval/shared.xml]: Used `oval_metadata` macro.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml (hunk 1 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/oval/shared.xml]: Used `oval_metadata` macro.

**Comment by mildas** [comment]
**Changes identified:**
Rule dconf_gnome_disable_automount:
 Node moved within OVAL check.
 Ansible remediation changed.
 Node deleted from OVAL check.
 Attribute value changed in OVAL check.
 Text changed in OVAL check.
 Template usage changed in ansible remediation.
 Deleted attribute from OVAL check.
Rule dconf_gnome_disable_automount_open:
 OVAL check is newly added.
 Ansible remediation newly added.
 Bash remediation is newly added.
Rule dconf_gnome_disable_autorun:
 OVAL check is newly added.
 Ansible remediation newly added.
 Bash remediation is newly added.
Profile ncp on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun added to ncp profile.
Profile stig on rhel7:
 Rule dconf_gnome_disable_automount_open, dconf_gnome_disable_autorun, dconf_gnome_disable_automount added to stig profile.

**Recommended tests to execute:**
 build_product rhel7
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_automount_open
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using ansible --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-rhel7-ds.xml dconf_gnome_disable_autorun
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml ncp
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-rhel7-ds.xml stig

**Comment by ggbecker** [comment]
> Oh, while running SST test suite, unfortunately none of ansible remediations seem to pass the test. Bash is ok.

I have addressed all the requests and fixed the ansible remediation (it was missing `no_extra_spaces: yes`). The OVAL regex doesn't expect any space between the `=` separator.

**Comment by redhatrises** [comment]
> > Looks good except for the duplicate STIGIDs. We should remove them, and ask for new ones.
> 
> I assume that it can take a while until they (DISA) split the rules and assign new STIG ids to them. So, do we want to wait for that? And how do we initiate this request to DISA?

Yes, we do want to wait for that. There should be no duplicate STIG IDs.

**Comment by ggbecker** [comment]
> > > Looks good except for the duplicate STIGIDs. We should remove them, and ask for new ones.
> > 
> > 
> > I assume that it can take a while until they (DISA) split the rules and assign new STIG ids to them. So, do we want to wait for that? And how do we initiate this request to DISA?
> 
> Yes, we do want to wait for that. There should be no duplicate STIG IDs.

Ok, also I remember seeing duplicated STIG ids in the project. I will try to find all of them and publish in a new issue.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+references:
+    cui: 3.1.7
+    nist: CM-7(a),CM-7(b),CM-6(a)
+    nist-csf: PR.AC-3,PR.AC-6
+    isa-62443-2013: 'SR 1.1,SR 1.13,SR 1.2,SR 1.4,SR 1.5,SR 1.9,SR 2.1,SR 2.6'
+    isa-62443-2009: 4.3.3.2.2,4.3.3.5.2,4.3.3.6.6,4.3.3.7.2,4.3.3.7.4
+    cobit5: APO13.01,DSS01.04,DSS05.03,DSS05.04,DSS05.05,DSS05.07,DSS06.03
+    iso27001-2013: A.11.2.6,A.13.1.1,A.13.2.1,A.6.2.1,A.6.2.2,A.7.1.1,A.9.2.1
+    cis-csc: 12,16
+    stig@rhel7: RHEL-07-020111
```
Comment (redhatrises) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml:42]: Remove this duplicate ID.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/rule.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+references:
+    cui: 3.1.7
+    nist: CM-7(a),CM-7(b),CM-6(a)
+    nist-csf: PR.AC-3,PR.AC-6
+    isa-62443-2013: 'SR 1.1,SR 1.13,SR 1.2,SR 1.4,SR 1.5,SR 1.9,SR 2.1,SR 2.6'
+    isa-62443-2009: 4.3.3.2.2,4.3.3.5.2,4.3.3.6.6,4.3.3.7.2,4.3.3.7.4
+    cobit5: APO13.01,DSS01.04,DSS05.03,DSS05.04,DSS05.05,DSS05.07,DSS06.03
+    iso27001-2013: A.11.2.6,A.13.1.1,A.13.2.1,A.6.2.1,A.6.2.2,A.7.1.1,A.9.2.1
+    cis-csc: 12,16
+    stig@rhel7: RHEL-07-020111
```
Comment (redhatrises) [inline linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/rule.yml:42]: Remove this duplicate ID.

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml (hunk 6 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_automount_open/rule.yml]: Done

File: linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/rule.yml (hunk 7 above)
Comment (ggbecker) [reply linux_os/guide/system/software/gnome/gnome_media_settings/dconf_gnome_disable_autorun/rule.yml]: Done

**Pull request merged**